rando_modules/modify_entrances.py

```diff
@@ -665,10 +665,11 @@ def adjust_rip_cheato_pricing(world_graph: dict, checks_in_logic:int):
     Returns the modified world graph itself with adjusted item check logic for
     the 11 item checks of Rip Cheato.
     """
-    base_cheato_edges = deepcopy([
-        edge for edge in world_graph["TIK_15/1"]["edge_list"]
-            if isinstance(edge["to"]["id"], str)
-    ])
+    base_cheato_edges = []
+    base_cheato_edges.extend([deepcopy(edge) for edge in world_graph["TIK_15/1"]["edge_list"] if isinstance(edge["to"]["id"], str)])
+    for x in "ABCDEFGHIJ":
```

Comment:
There's a giftK, shouldn't it be x in "ABCDEFGHIJK"?

Reply:
We only need to adjust logic for outgoing edges, and since GiftK is the last item it has no outgoing edge we'd have to adjust.
The first edge that leads from TIK_15/1 to TIK_15/GiftA is added in the former step on L669